CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v0.2.0
+
```

Comment:
Usually there is a line showing the diff.  See https://github.com/openfga/js-sdk/blob/cbf282f1666991d2959129292ad6a26369360e63/CHANGELOG.md?plain=1#L5

Reply:
Added.

Reply:
It should be a diff between 0.2.0 and 0.1.0

Reply:
Done.